packages/shared/src/styles/base.css

```diff
@@ -36,6 +36,15 @@ body {
   }
 
   overflow-x: hidden;
+
+  @screen mobileL {
```

Comment:
This helps fixing the footer navbar. One of the issues is that on all pages, you must add extra padding due to the fact that we are using `fixed` property. With the properties below, this will make using `sticky` work.

Reply:
This feels very heavy to add to generic body tag 🙈 
I'm a bit worried it will have some implication on somethings we haven't tested.

Reply:
Yeah, same sentiments. However, I think it is best to use `sticky` on the footer navbar as it is problematic across all pages which this fixes. Also why I am immediately setting the values back to its original state after mobile.

Reply:
There is no way we can only target that element vs the whole body tag?

Reply:
Let me check, though as far as I tested yesterday, those were the requirements. Let me check if I move the elements 1 level deeper.

Reply:
I checked having it inside `MainLayout` component but still needed the CSS properties defined here. With that, I've created a list to check on all 3 screen sizes.

Maybe we should store this list somewhere to make it easier next time we have something similar to this in the future.
![Screenshot 2024-04-04 at 5 35 04 PM](https://github.com/dailydotdev/apps/assets/13744167/1465c5a0-c514-4d6d-92ee-ff21823ef480)